### ISSUE TITLE
Help, please

### ISSUE THREAD

**Issue by henriqueyang** [adam-hanna/goalSeek.js#3]
First, happy new year!

Could you help me in my case, where I need an intended value of pmt (periodic paymount) by changing the rate (float number)? It seems to me that this goalSeek function doesn't handle this case well. 

My code: https://repl.it/@henriqueyang/goal-seek-pmt

Thanks!

**Comment by adam-hanna** [comment]
@henriqueyang sure, I'll gladly help. Let me take a look...

**Comment by adam-hanna** [comment]
@henriqueyang  Ok, I took a look.

First, let me apologize, my code is a mess and needs to be updated.

Second, I should have added a feature called "max jump" that doesn't allow the inputs to be moved any more than X. The problem was, your function was very sensitive to changes to inputs, and couldn't handle negative interest rates. The goal seek function was making these huge swings and sometimes going negative, which was causing it to err.

Anyway, I made some minor changes. here:
https://repl.it/repls/DangerousExcitedDriver
